src/main/java/com/the/dark/side/crew/fejsbuk/controller/UserController.java

```diff
@@ -0,0 +1,28 @@
+package com.the.dark.side.crew.fejsbuk.controller;
+
+import com.the.dark.side.crew.fejsbuk.model.User;
+import com.the.dark.side.crew.fejsbuk.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequiredArgsConstructor
+public class UserController {
+
+    @Autowired
```

Comment:
autowired is not necessary, spring will inject userService using available constructor